src/graph/product.py

```diff
@@ -311,8 +320,8 @@ def construct_product(self):
                                                                         _v_a_node,
                                                                         _v_prod_node,
                                                                         action=auto_action,
-                                                                        obs=ap,
-                                                                        pref=pref)
+                                                                        obs=ap,)
+                                                                        # pref=pref)
```

Comment:
I'll be needing this line. So we need to think how we construct this piece line of code